Sources/SwiftDocCUtilities/Action/Actions/Convert/ConvertAction.swift

```diff
@@ -200,7 +203,8 @@ public struct ConvertAction: Action, RecreatingContext {
         inheritDocs: Bool = false,
         experimentalEnableCustomTemplates: Bool = false,
         transformForStaticHosting: Bool,
-        hostingBasePath: String?
+        hostingBasePath: String?,
+        temporaryDirectory: URL
```

Comment:
Nit: this is slightly odd as parameter with no default value for `ConvertAction`. I wouldn't expect clients to need to provide a temporary directory, since using a temporary directory is an implementation detail of `ConvertAction`. Should we set a default value of `FileManager.default.temporaryDirectory`?

Reply:
I would prefer to not provide a default value for this argument to avoid mistakes where a test would forget to provide a custom value.